ui/public/i18n/en/translation.json

```
@@ -49,7 +50,10 @@
     "network_tooltip": "Make sure this network does not overlap with the units' networks. The field will be read-only after the first configuration",
     "netmask_tooltip": "Do not change this value unless you know what you are doing. The field will be read-only after the first configuration",
     "netmask_helper": "The netmask determines the size of the VPN network. Use only class C networks",
-    "password_placeholder": "Password already set, enter a new one to change it"
+    "password_placeholder": "Password can be modified from the controller webapp",
+    "password_information_title": "Password information",
+    "password_information_description": "The password is diplayed only once, please store it in a safe place",
```

There is a typo on `diplayed`. I also propose a little improvement.

```suggestion
    "password_information_description": "The default administrator password is displayed only once, please store it in a safe place",
```